source/scss/style.scss

```diff
@@ -96,5 +96,5 @@
 // Images
 @import "components/images";
 
-// Brand Colors
+/// Brand Colors
```

Comment:
If i'm correct, we need to add three slashes to our code so that Cssdocs detects our documentation, right? If that's so, please add this info to the README file.

Reply:
Hey, thank you for considering this PR. 🙏

Yes, for normal comments you continue with double slash:
```
// This is a comment and can be placed anywhere, where comments are allowed.
```
```
/// This is a description in SassDocs and is usually attacked above the subject you want to annotate and describe.
.my-subject-class {
  // another comment
}
```
The three slashes, a description, alone will already make this class available inside the documentation. 

For annotations and documentations you work with 3 or 4 (at the beginning of the page. See below.)

[See the documentation about annotations in Sassdocs](http://sassdoc.com/annotations/).